content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -200,9 +200,9 @@ and automatically adds a default storage class to them.
 This way, users that do not request any special storage class do not need to care about them at all and they
 will get the default one.
 
-This admission controller does not do anything when no default storage class is configured. When more than one storage
-class is marked as default, it rejects any creation of `PersistentVolumeClaim` with an error and an administrator
-must revisit their `StorageClass` objects and mark only one as default.
+This admission controller does nothing when no default `StorageClass` exists. When more than one storage
+class is marked as default, and you then create a PersistentVolumeClaim with no storageClassName set, 
```

Comment:
(nit)
```suggestion
class is marked as default, and you then create a PersistentVolumeClaim with no `storageClassName` set, 
```

Reply:
Does the defaulting at admission time happen even if `volumeName` is set? If not, maybe we should mention that.

Reply:
What I experienced - the admission controller assigned the most recent class to the PVC, but the PVC was in a pending status until a volume with a name matching the `volumeName` on the PVC and the `storageClassName` on both the PV and PVC were matching was statically provisioned.

Reply:
Consider this...
I have a PV statically provisioned, and I know its name. Now I'm creating a PVC to bind to that PV. In this PVC, I'm not specifying a `storageClassName` but I'm setting the `volumeName` to express my intent. This is a pretty valid (and a common) use case.
Now, re-read your revision. Does it still look correct?

Reply:
@sftim, @tengqm - https://github.com/kubernetes/website/pull/48200#discussion_r1790560045 has been included in the revision.